src/deploy/scope_managementgroup.ts

```diff
@@ -36,6 +36,7 @@ export async function DeployManagementGroupScope(azPath: string, region: string,
             },
             stdout: (data: BufferSource) => {
                 commandOutput += data.toString();
+                core.info(commandOutput);
```

Comment:
This could expose secrets in the logs. especially if ARM template has some output.

Reply:
Hey @bishal-pdMSFT. Good point! However, the commandOutput is just an output/buffer from the usual `az deployment <group/mg/sub>` cli commands like shown here

https://docs.microsoft.com/en-us/cli/azure/deployment/group?view=azure-cli-latest#az_deployment_group_create

There is possibility of secrets being exposed, but that is a responsibility of the end user who writes the arm template. For example, putting type `securestring` instead of `string` so that when the output goes to standard out, it does not expose the secret.

Reply:
One additional suggestion is to start using dependabot if you don't already

https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/#:~:text=To%20enable%20Dependabot%20version%20updates%2C%20look%20for%20a,version%20updates%20is%20done%20via%20the%20configuration%20file.

This is to help you automatically manage updating versions in your npm packages

Reply:
Dependabot is already active on this repo. Thanks!

Reply:
@jontreynes To add to above, the action already adds the template output to environment variable so that next step can consume it. 
I do not see a concrete reason for printing the output. We could do that for debug purpose if debug flag is set to true.